main.go

```diff
@@ -111,6 +113,14 @@ func NewGame(g *Game) {
 		}
 	}
 
+	// Music
+	const sampleRate int = 44100 // assuming "normal" sample rate
```

Comment:
You will quickly discover if the sample rate didn't match because Ebitengine will resample it on the fly and the game's performance will drop noticeably, especially on a weaker computer it will stutter.